models/quality_measures/intermediate/cbe0055_diabetes_eye_exam/quality_measures__int_cbe0055_denominator.sql

```diff
@@ -50,11 +50,19 @@ with visit_codes as (
           patient_id
         , procedure_date as min_date
         , procedure_date as max_date
-    from {{ref('quality_measures__stg_core__procedure')}} proc
+    {% if target.type == 'fabric' %}
+        from {{ref('quality_measures__stg_core__procedure')}} "proc"
+    {% else %}
+        from {{ref('quality_measures__stg_core__procedure')}} proc
+    {% endif %}
```

Comment:
I noticed there are a lot of these "if" statements for a table alias. Can we just change the alias to something else like "procedure"?

Reply:
I updated those to 'procedures' and 'procs' so they don't needs the quotes anymore.